.github/workflows/codecov.yml

```diff
@@ -9,11 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
IIRC `dtolnay/rust-toolchain` offers a `stable` version, which makes it possible to ditch the separate `toolchain` argument, would be a nice improvement.

Reply:
Yes, I've just changed that.